### ISSUE TITLE
Script works but doesn't take effect

### ISSUE THREAD

**Issue by fedesuarez79** [plyint/nvidia-overclock.sh#1]
Great script works like a charm! however my rtx3070s worked by changing [3] by [4] in the commands like:
log "$(nvidia-settings -c :0 -a '[gpu:0]/GPUGraphicsClockOffset[4]=-500' -a '[gpu:0]/GPUMemoryTransferRateOffset[4]=2400')"

As an additional note, in order to get the startx running lightdm must be purged previously...

**Comment by ahnick** [comment]
Yes, depending on your graphics card it turns out the array value may be different.  I probably won't update the script to do any kind of auto detection or removal of lightdm, but I can add these to the README as additional pieces of documentation.

**Comment by fedesuarez79** [comment]
perfect! great work and thanks for sharing again!

**Comment by iopq** [comment]
I had to do this:

`nvidia-settings -a [gpu:0]/GPUMemoryTransferRateOffsetAllPerformanceLevels=2000 -a [gpu:0]/GPUGraphicsClockOffsetAllPerformanceLevels=-300`

otherwise it just errored out on my RTX 2060
